packages/@coorpacademy-components/src/atom/radio-with-title/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useCallback, useState} from 'react';
+import PropTypes from 'prop-types';
+import {noop, uniqueId} from 'lodash/fp';
+import Title from '../title';
+import style from './style.css';
+import {RadioWithTitleProps} from './types';
+
+const RadioWithTitle = (props: RadioWithTitleProps) => {
+  const {
+    title,
+    name,
+    checked,
+    onChange = noop,
+    'data-name': dataName,
+    'aria-label': ariaLabel
+  } = props;
+  const [_checked, setChecked] = useState(checked);
+
+  const idRadio = uniqueId('input-radio-');
+  const handleChange = useCallback(
```

Comment:
handleChange event is not covered, `input` should be triggered on a unit test to cover it